src/core/events.ts

```diff
@@ -48,6 +48,7 @@ export const REDIRECT = 'redirect';
 
 // error events
 export const ERROR_BRIDGE = 'error:bridge';
+export const ERROR_FETCH_ACTION = 'error:fetch_action';
```

Comment:
I wanted to group all application errors under one namespace rather than have an error for each action, but also I was copying the bridge error. I feel like having the error namespace is better than having errors scattered throughout other namespaces